api/v1beta1/vmcluster_types.go

```diff
@@ -729,6 +729,23 @@ type VMBackup struct {
 	// that are generated as a result of StorageSpec objects.
 	// +optional
 	VolumeMounts []v1.VolumeMount `json:"volumeMounts,omitempty"`
+
+	// Restore Allows to enable restore options for pod
+	// Read more: https://docs.victoriametrics.com/vmbackupmanager.html#restore-commands
+	// +optional
+	Restore *VMRestore `json:"restore,omitempty"`
+}
+
+type VMRestore struct {
```

Comment:
WDYT about adding boolean field -  `EnableRestoreCommands` to the `type VMBackup struct`?

 It should simplify logic a bit.

Reply:
This level of nesting was added in order to support the following future use-cases (without making breaking changes):
- ability to add separate extraArgs/extraEnvs for restore init container
- ability to run restore from point defined in k8s manifest (for example to create cluster and restore it from certain backup on start)
so that it will be possible to use something like:
```
restore:
  onStart:
    fromBackup: "some/backup/ref"
```

Reply:
If this seems to be redundant at this point please let me know - I'd be happy to simplify structs nesting here :)